docs/operation/index.rst

```diff
@@ -10,16 +10,14 @@ do not have a working MetalK8s_ setup.
 
 .. toctree::
 
-   bootstrap_backup_restore
-   enable_ip_ip
-   preparation
    solutions
    upgrade
    downgrade
+   disaster_recovery/index
    changing_node_hostname
    volume_management/index
-   account_administration
+   account_administration/index
    cluster_and_service_configuration
-   cluster_monitoring
+   cluster_monitoring/index
    metalk8s-utils
-   troubleshooting
+   troubleshooting/index
```

Comment:
Personally, I would suggest the following order to be confirmed (@gdemonet @xaf-scality)

cluster_monitoring/index
account_administration/index
cluster_and_service_configuration
volume_management/index
upgrade
downgrade
disaster_recovery/index
solutions
changing_node_hostname
metalk8s-utils
troubleshooting/index

Reply:
Fine by me :+1: